R/dummy_multi_label.R

```diff
@@ -0,0 +1,309 @@
+#' multi Label Dummy Variables Creation
+#'
+#' `step_dummy_multi_label` creates a *specification* of a recipe
+#'  step that will convert multiple nominal data (e.g. character or factors)
+#'  into one or more numeric binary model terms for the levels of
+#'  the original data.
+#'
+#' @inheritParams step_center
+#' @inherit step_center return
+#' @param ... One or more selector functions to choose which variables will be
+#'  used to compute the components. See [selections()] for more details.
+#' @param role For model terms created by this step, what analysis
+#'  role should they be assigned?. By default, the function assumes
+#'  that the binary dummy variable columns created by the original
+#'  variables will be used as predictors in a model.
+#' @param threshold A numeric value between 0 and 1 or an integer greater or
```

Comment:
This really bit us for something similar that we did with xgboost. I'd say let's just use the fractional value.